pennylane/data/data_manager/__init__.py

```diff
@@ -152,28 +202,26 @@ def list_datasets() -> dict:
     Return:
         dict: Nested dictionary representing the directory structure of the hosted datasets.
 
+    .. seealso:: :func:`~.load_interactive`, :func:`~.list_attributes`, :func:`~.load`.
+
     **Example:**
 
     Note that the results of calling this function may differ from this example as more datasets
     are added. For updates on available data see the `datasets website <https://pennylane.ai/qml/datasets.html>`_.
 
-    .. code-block :: pycon
-
-        >>> qml.data.list_datasets()
-        {'qchem': {'H2': {'6-31G': ['0.5', '0.54', '0.58', ... '2.02', '2.06', '2.1'],
-                          'STO-3G': ['0.5', '0.54', '0.58', ... '2.02', '2.06', '2.1']},
-                   'HeH+': {'6-31G': ['0.5', '0.54', '0.58', ... '2.02', '2.06', '2.1'],
-                            'STO-3G': ['0.5', '0.54', '0.58', ... '2.02', '2.06', '2.1']},
-                   'LiH': {'STO-3G': ['0.5', '0.54', '0.58', ... '2.02', '2.06', '2.1']},
-                   'OH-': {'STO-3G': ['0.5', '0.54', '0.58', ... '0.94', '0.98', '1.02']}},
-        'qspin': {'Heisenberg': {'closed': {'chain': ['1x16', '1x4', '1x8'],
-                                            'rectangular': ['2x2', '2x4', '2x8', '4x4']},
-                                 'open': {'chain': ['1x16', '1x4', '1x8'],
-                                        'rectangular': ['2x2', '2x4', '2x8', '4x4']}},
-                  'Ising': {'closed': {'chain': ['1x16', '1x4', '1x8'],
-                                        'rectangular': ['2x2', '2x4', '2x8', '4x4']},
-                            'open': {'chain': ['1x16', '1x4', '1x8'],
-                                    'rectangular': ['2x2', '2x4', '2x8', '4x4']}}}}
+    >>> available_data = qml.data.list_datasets()
+    >>> available_data.keys()
+    dict_keys(["qspin", "qchem"])
+    >>> available_data["qchem"].keys()
+    dict_keys(["H2", "LiH", ...])
+    >>> available_data['qchem']['H2'].keys()
+    dict_keys(["6-31G", "STO-3G"])
```

Comment:
```suggestion
    dict_keys(["CC-PVDZ", "6-31G", "STO-3G"])
```
Added the extra basis set now that it's available. :+1: